lib/ansible/modules/database/mysql/mysql_variables.py

```diff
@@ -44,6 +50,12 @@
 - mysql_variables:
     variable: read_only
     value: 1
+
+- name: Set read_only variable persistently
+- mysql_variables:
+    variable: read_only
+    value: 1
+    persist: yes
```

Comment:
```suggestion
    mode: persist
```
regarding the [documentation](https://dev.mysql.com/doc/refman/8.0/en/set-variable.html) maybe we could change ```persist``` option to something like:
```
mode:
  description: ...
  type: str
  choices: ['global', 'persist', 'persist_only']
  default: 'global'
```
(i guess 'session' support does not make sense)

Reply:
because of how it all works, session support doesn't currently make sense, but would in theory be cool if you could somehow re-use the same mysql session for a later task, so you could set session variables for an ansible play this way